service/signinglog/actions.go

```diff
@@ -37,7 +37,6 @@ type ListResponse struct {
 	ErrorSubcode string                 `json:"error_subcode"`
 	ErrorMessage string                 `json:"message"`
 	SigningLog   []datastore.SigningLog `json:"logs"`
-	Total        int                    `json:"total"`
```

Comment:
I don't understand this change,  my suggestion talked about SigningLog, what I was thinking was more:

```
@@ -110,6 +103,7 @@ type SigningLog struct {
	Created      time.Time `json:"created"`
	Revision     int       `json:"revision"`
	Synced       int       `json:"synced"`
	Total        int       `json:"total_count"`
}
```

remove the `json:"total_count"` tag there